README.md

```diff
@@ -116,7 +116,9 @@ For profiling with eBPF, kernel headers must be accessible from within the conta
 The command above mounts both of these directories.
 
 ## Running as an executable
-Run the following to have gprofiler running continuously, uploading to Granulate Performance Studio:
+First, check if gProfiler is already running - run `pgrep gprofiler`. You should see 3 PIDs (staticx bootloader, PyInstaller and gProfiler's Python itself). If no processes match, then gProfiler is not running.
+
```

Comment:
The doc here doesn't seem to follow the "happy flow", and is overall confusing. Since this is the doc for running the gProfiler, most of the time for the first time, I would replace `You should see 3 PIDs` with `The command should show no output. If you see 3 PIDs, it means the gProfiler is already running, and you could stop it by running...`

Reply:
K will improve. I might change the "numerous" PIDs part altogether, because I'm considering to skip staticx for Aarch64 since it's messed up (see https://github.com/JonathonReinhart/staticx/issues/181#issuecomment-915616425, and now I'm getting:
```
 => ERROR [build-stage 43/43] RUN if [ $(uname -m) != "aarch64" ]; then source scl_source enable devtoolset-8 llvm-toolset-7; fi && libs=$(./scripts/list_needed_libs.sh) && staticx $libs dist/gpro  0.9s
------
 > [build-stage 43/43] RUN if [ $(uname -m) != "aarch64" ]; then source scl_source enable devtoolset-8 llvm-toolset-7; fi && libs=$(./scripts/list_needed_libs.sh) && staticx $libs dist/gprofiler dist/gprofiler:
#80 0.803 ldd failed: ldd: exited with unknown exit code (139)
------
error: failed to solve: executor failed running [/bin/sh -c if [ $(uname -m) != "aarch64" ]; then source scl_source enable devtoolset-8 llvm-toolset-7; fi && libs=$(./scripts/list_needed_libs.sh) && staticx $libs dist/gprofiler dist/gprofiler]: exit code: 1
```
(`ldd` gets SIGSEGV...))

Reply:
I decided to skip staticx